mordant/src/jvmMain/resources/META-INF/proguard/mordant.pro

```diff
@@ -1,4 +1,7 @@
 # Keep rules for those who are using ProGuard.
 
 -keep class com.sun.jna.** { *; }
--keep class * implements com.sun.jna.** { *; }
\ No newline at end of file
+-keep class * implements com.sun.jna.** { *; }
+-keepattributes *Annotation*
```

Comment:
This is a very broad rule that keeps a lot of things that I suspect are not needed. Usually it's recommended to do something like `RuntimeVisibleAnnotations,RuntimeVisibleParameterAnnotations,RuntimeVisibleTypeAnnotations,AnnotationDefault` instead which allows the removal of runtime-invisible annotations. The full list of attributes is available here: https://www.guardsquare.com/manual/configuration/attributes

Reply:
Awesome, thanks for the insight!